gift_compassion/models/sponsorship_gift.py

```diff
@@ -201,7 +201,6 @@ def get_sponsorship_gifts(self):
     @api.depends(
         "invoice_line_ids",
         "invoice_line_ids.parent_state",
-        "invoice_line_ids.price_subtotal",
```

Comment:
You should add those two new dependencies instead: `invoice_line_ids.credit`, `invoice_line_ids.debit`